css/values.css

```diff
@@ -0,0 +1,336 @@
+	font-weight: 700;
+	line-height: 50px;
+}
+
+.support a {
+	width: fit-content;
+}
+
+.support-text {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+
+/* layout */
+.support .wrapper {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-med);
+}
+
+/* heading and heart layout*/
+.support-heading {
+	display: flex;
+	gap: var(--spacing-sm);
+}
+
+.support-heading img {
+	width: var(--spacing-med);
+}
+
+/* support un images */
+.support-img-container {
+	display: flex;
+	flex-wrap: wrap;
+	gap: var(--spacing-sm);
+}
+
+.support-img-container img {
+	width: fit-content;
+	max-height: 100px;
+}
+/* 
+===================================
+Empowering 
+===================================
+*/
+/* using styled img container component in global.css */
+
+.empowering .wrapper {
+	flex-direction: row-reverse;
+}
+
+/* 
+===================================
+Media Queries (overriding styles for different screen sizes)
+===================================
+*/
+
+/* Tablet and smaller 1024px*/
+@media (max-width: 1024px) {
+	/* 
+===================================
+Values
+===================================
+*/
+	.values-img-text .img-mobile {
+		display: block;
+		visibility: visible;
+	}
+	/* layout to colum */
+	.values-row,
+	.values-img-text {
+		flex-direction: column;
+	}
+
+	.values-card {
+		width: 100%;
+		padding: var(--spacing-sm);
+		min-height: 300px;
+	}
+
+	/* Img + Text Card styles */
+	.values-img-text {
+		background: none;
+		padding: 0;
+		display: flex;
+	}
+
+	/* images treated like cards */
+	.values-img-text .text,
+	.values-img-text img {
+		border-radius: var(--border-radius);
+		min-height: 300px;
+	}
+
+	.values-img-text .text {
+		padding: var(--spacing-sm);
+		justify-content: center;
+		width: 100%;
+	}
+	.values-img-text img {
+		object-fit: cover;
+	}
+
+	/* value card text backgrounds */
+	.sustainability .text {
+		background: #0f0f0f;
+	}
+	.authenticity .text {
+		background-color: var(--color-amaranth);
+	}
+	.education .text {
+		background-color: var(--color-blue);
+	}
+	.innovation .text {
+		background-color: var(--color-teal);
+	}
+
+	/* 
+===================================
+Empowering
+===================================
+*/
+	.empowering {
+		padding: var(--spacing-xxl) 0 0 0;
+	}
+	.empowering .wrapper {
+		flex-direction: column;
+		width: 100%;
+		padding: 0;
+	}
+
+	/* removing absolute positioned graphic */
+}
+
+/* Tablet and smaller 768px */
+@media screen and (max-width: 768px) {
+	/* 
+===================================
+Hero
+===================================
+*/
+	/* mobile view + layout */
+	.hero {
+		background: none;
+	}
+
+	.hero .wrapper {
+		justify-content: flex-start;
+	}
+
+	.hero-text-container {
+		width: 100%;
+		text-align: center;
+	}
+}
+
+/* Mobile Only 470px */
+@media screen and (max-width: 470px) {
+	/* 
+===================================
+Mobile Fonts
+===================================
+*/
+	.support h2,
+	.empowering h2 {
```

Comment:
The "Empowering" title centers on mobile. Check padding as well.

<img width="766" alt="Screen Shot 2024-01-11 at 3 03 09 PM" src="https://github.com/indigenousfriends/itlct-website-development/assets/704697/15d4e3ce-f92b-434a-be20-3d6f07e77b25">

Reply:
Thank you! addressed text and spacing for this section